crates/shared/src/http_solver/model.rs

```diff
@@ -193,7 +193,7 @@ pub enum Score {
     /// To have more flexibility, the protocol score can be tweaked by the
     /// solver by providing a multiplication factor. Expected value: [0,
     /// inf], 0 being 100% discount, 1 being the same as protocol score
-    MulFactor(f64),
+    ScoreFactor(f64),
```

Comment:
Sorry for the double nit 🙈 - Here I would actually do:
```suggestion
    #[serde(rename = "scoreFactor")]
    Factor(f64),
```

My rationale is that `Score::Factor` in Rust is less repetitive than `Score::ScoreFactor`, but that in the JSON it should be called `scoreFactor`).

This is very much a "uber-nit", as this only appears in the HTTP solver DTO (i.e. `model.rs`), so feel free to ignore.

Reply:
Fixed. Just with discount name